app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```
@@ -69,6 +71,7 @@ $gem-c-accordion-bottom-border-width: 1px;
     }
 
     &:after {
+      @include prefixed-transform(0, -45deg);
```

I like this mixin - can it be used with keyword arguments to make it clearer what it does?

```suggestion
      @include prefixed-transform($rotate: -45deg);
```